build.ps1

```diff
@@ -266,6 +268,8 @@ try {
         BuildPackage provisioning\transport\http\src "Provisioning Transport for HTTP"
         BuildPackage provisioning\transport\mqtt\src "Provisioning Transport for MQTT"
         BuildPackage provisioning\service\src "Provisioning Service Client SDK"
+		BuildPackage digitaltwin\device\src "Digital Twin Device Client SDK"
```

Comment:
nit (unimportant right now): extra space

Reply:
Sure, I can fix this